packages/headless/src/app/engine.ts

```diff
@@ -65,6 +68,8 @@ export interface CoreEngine<
   store: Store;
   /**
    * A function for headless to call to retrieve a refreshed access token.
+   *
+   * @deprecated - Calling this function directly is not needed because Headless handles token renewal internally. The function will be removed in the next major version.
```

Comment:
Hum, I'm not sure it was ever the case ? 🤔  Wasn't that just a config option ?

Reply:
The configured function is being wrapped by logic to prevent infinite loops and then is exposed as part of the `HeadlessEngine` public api. The following is currently possible:

```
new HeadlessEngine(...options...).renewAccessToken();
```